homeassistant/components/sensor/enphase_envoy.py

```diff
@@ -0,0 +1,108 @@
+"""
+Support for Enphase Envoy solar energy monitor.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/sensor.enphase_envoy/
+"""
+import logging
+import voluptuous as vol
+
+from homeassistant.helpers.entity import Entity
+from homeassistant.components.sensor import PLATFORM_SCHEMA
+import homeassistant.helpers.config_validation as cv
+from homeassistant.const import (CONF_IP_ADDRESS)
+from homeassistant.const import (CONF_MONITORED_CONDITIONS)
```

Comment:
Group imports from the same module. I use `isort` to do this automatically for new modules. Most advanced editors have plugins that can do this.